### ISSUE TITLE
Idea - remaping symbols to new ones

### ISSUE THREAD

**Issue by skynet01** [oodesign/merge-duplicate-symbols#121]
Would be great to have an option to map symbols from one type to another. Say i have a file that uses one library of icons and i need to remap all icons to new library. Would be awesome if i could do that :) 

Thanks and keep up great work

**Comment by oodesign** [comment]
Hi @skynet01 ,

Yup, that would be awesome!
However, how do you envision this mapping done?

If it's by name (if icon names in old and new libraries match) you can already use Merge Duplicates (or Sync with Library) for that.

If it's on ID (this may only happen if you actually duplicate the icon library and change the icons in the new one, as internal symbols IDs would be the same) you can use the Automate plugin (Automate > Library > Replace library).

But if it's not on name or ID, what other attributes/elements/concepts could be used to remap them? 🤔

Thanks for your feedback and help! 🥰

**Comment by skynet01** [comment]
Thanks for the tip on the Automate plugin, this thing is great! I really like your UI for remapping similar entities (you just select items from the list on the left then you just select which one to keep). We can use the same approach here. The user gets a list of all their symbols on the left, then you can click on each one and by default, we can match them by name with an option to override. 

Here is a crude example: (ignore that this is for styles, I just want to show that you can do more than one substitution ) 
![image](https://user-images.githubusercontent.com/8117544/98025974-ab9fb500-1dc7-11eb-92d0-ae0155c483a3.png)

The problem with the Automate plugin or doing manual replacement is that it takes forever to go one by one. With your plugin, I could remap all of my 50 icons in 10 minutes since everything is in one place (even if I have to manually select them from a drop-down one by one)

We might need other options like keeping the same size, apply the same color style, etc.. (if the user selects a different symbol size).. but those are bonuses :)

**Comment by oodesign** [comment]
Oh, I see! 

So as far as I understand, you'd like to be able to pick manually the symbols you'd like to "merge" (replace), in a similar way the "Merge duplicate --- (from list)" option does?

That'd be a nice addition, for sure. The only concern we have with that is that it'd be still quite slow, as those "pick from list" options don't offer the ability to do several merges/replacements in a row... but that may also be an improvement 🤔.

We'll be gathering feedback around both improvements, and considering including them in Merge Duplicates. I'll make sure to keep you updated on it.

Thanks a lot for your feedback!

**Comment by skynet01** [comment]
Yeah .. i know you cant do multiple at once. That would be a time saver for sure as i have to open plugin for each one...  

I am ok to wait if there is a progress bar somewhere. I can just go do something else :) What I don't want to do is spend hours manually going through each symbol how I have to do now :)
